buildbot-conf/cordova.conf

```diff
@@ -229,6 +229,8 @@ plugins_cleanup_steps = [
     NPMInstall(command=['shelljs'], what='shelljs'),
     RMRF('~/.cordova/*', description='removing cache', haltOnFailure=False),
     RMRF(BASE_WORKDIR + '/*', description='cleaning workspace', haltOnFailure=False),
+    RMRF(BASE_WORKDIR + '/*', description='cleaning workspace again', haltOnFailure=False),
+    RMRF(BASE_WORKDIR + '/*', description='cleaning workspace yet again', haltOnFailure=True),
```

Comment:
I think user of Buildbot can't fix issue inside Windows filesystem. So it is unnecessary to show that "something that still needs to be fixed properly" to him.

Though we haven't seen the issue for `~/.cordova/*` it could happen. I assume that the cause of the issue is Windows filesystem and so `~/.cordova/*` and workspace folder have equal probability to have this issue.

Also other developer could use `RMRF` function for removing other folders. It would be good to provide a solid solution to remove folder.